### DISCUSSION TITLE
UI/UX Inconsistency

### DISCUSSION THREAD

**Discussion by iamsantanubanerjee** [thuliteio/doks#660] [Ideas]
The drop down card does not autoadjust in the doks child theme. Below is an example:

![Inconsistency](https://user-images.githubusercontent.com/59969215/153178414-0db90618-ebad-48e7-9dc6-993d7e08281d.png)

![Inconsistency_dark](https://user-images.githubusercontent.com/59969215/153178743-59c0acb4-b94a-459d-b828-a2c3338a1b47.png)

![Inconsistency3](https://user-images.githubusercontent.com/59969215/153179102-4d22b26c-1cad-4250-a173-c2ea967c1542.png)

**Comment by h-enk** [comment]
Thanks for reporting this — I've created an issue: https://github.com/h-enk/doks/issues/665

**Reply by h-enk** [reply]
https://github.com/h-enk/doks/issues/665#issuecomment-1039221667
